.github/workflows/scripts/lint-changed.sh

```diff
@@ -32,6 +32,9 @@ GITHUB_PR_NUMBER=$1
 # Tracker: https://github.com/eslint/eslint/issues/15010
 ESLINT_SUPPORTED_EXT=(js jsx ts tsx)
 
+# Excluding files that adhere to the specified patterns from the list of supported files.
+SUPPORTED_EXT_REMOVING_PATTERN=("java/apps/**/js/*.js")
```

Comment:
Let's rename this to
`SUPPORTED_EXT_REMOVING_PATTERN` -> `PATHS_TO_EXCLUDE`

And exclue all `.js` files in `java/apps`.

```suggestion
PATHS_TO_EXCLUDE=("java/apps/**/*.js")
```

Reply:
https://github.com/wso2/identity-apps/pull/3981/commits/3dbd94ccd8112770bbc586a136fae02267c875fe